next/app/lib/components/Navbar.tsx

```diff
@@ -0,0 +1,109 @@
+"use client";
+
+import React from "react";
+import { Routes } from "../constants";
+import Link from "next/link";
+import { usePathname } from "next/navigation";
+import { User } from "@auth/core/types";
+import { FaAngleDown, FaAngleUp } from "react-icons/fa";
+import { keycloakSignOut } from "../actions/keycloak";
+import { Role } from "@/prisma/generated/client";
+
+export interface INavbarOption {
+  label: string;
+  route: string;
+  roles: Role[];
+}
+
+export interface INavbarProps {
+  user: User;
+}
+
+/** Client Component used for navigation */
+export const Navbar: React.FC<INavbarProps> = ({ user }) => {
+  const pathname = usePathname();
+  const [showUserDropDown, setShowUserDropDown] = React.useState(false);
+
+  const navItems: INavbarOption[] = [
+    { label: "Home", route: Routes.Home, roles: [Role.ZEVA_USER] },
```

Comment:
I think the "Home" link should be unrestricted, because not every user may have the ZEVA_USER role or the ADMINISTRATOR role (the ZEVA_USER role is given only to bceid users, and so a government user may have only the ENGINEER_ANALYST role).